src/lib/operators/export_binary.cpp

```diff
@@ -127,6 +129,14 @@ std::shared_ptr<const Table> ExportBinary::_on_execute() {
     _write_chunk(table, ofstream, chunk_id);
   }
 
+  if (table->is_partitioned()) {
+    _write_partitioning_header(table, ofstream);
+
+    for (PartitionID partition_id{0}; partition_id < table->get_partition_schema()->partition_count(); partition_id++) {
+      _write_partition(table, ofstream, partition_id);
```

Comment:
Small thing: `_write_partition` could be interpreted as the method that actually writes data. `_write_partition_info`?